modules/AlphaWalletFoundation/AlphaWalletFoundation/Types/TokenProviderType.swift

```diff
@@ -146,6 +146,27 @@ public class TokenProvider: TokenProviderType {
         }
     }
 
+    fileprivate static func shouldRetry(_ error: Error) -> Bool {
```

Comment:
Should be `shouldRetryError(_ error: Error)` or `shouldRetry(error: Error)`. More idiomatic?